File: Java/Q26-Remove-Duplicates-from-Sorted-Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int p=-1;
        HashMap<Integer,Integer> map=new HashMap();
        for(int i=0;i<nums.length;i++){
            if(map.get(nums[i])==null){
                map.put(nums[i],1);
                nums[++p]=nums[i];
            }
        }
        return p+1;
    }
}

File: Typescript/Q26-Remove-Duplicates-from-Sorted-Array.ts
function removeDuplicates(nums: number[]): number {
    let p=-1;
    let map=new Map();
    for(let i=0;i<nums.length;i++){
        if(map.get(nums[i])==undefined){
            map.set(nums[i],1);
            nums[++p]=nums[i];
        }
    }
    return p+1;

};